crates/wallet/tests/wallet.rs

```diff
@@ -4116,3 +4116,40 @@ fn test_insert_tx_balance_and_utxos() {
     assert!(wallet.list_unspent().next().is_none());
     assert_eq!(wallet.balance().total().to_sat(), 0);
 }
+
+#[test]
```

Comment:
Just in case, maybe we should create a test that also recovers a single descriptor wallet from persistence.

Reply:
Done in 27da178c4a5581fca9ca1e7706ec47629ba65b4c